.eslintrc.js

```diff
@@ -14,9 +14,14 @@ module.exports = {
     project: true,
     tsconfigRootDir: __dirname
   },
-
+  plugins: [
+  ],
```

Comment:
is this required to be here if it's empty?

Reply:
removed.